app/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ sdk: ">=1.19.0 <2.0.0"
 dependencies:
   appengine: '^0.4.0'
   args: '^0.13.7'
+  ducene: '^0.4.0'
```

Comment:
This is a 3rd-party package and every upgrade can potentially cause us to pull in malicious code. The only other package is `uuid` I think.

Maybe put them into their own section and add a comment that these are 3rd-party packages we are relying on.

We could also think about pin-pointing the version exactly instead of having a loose constraint.